src/content/docs/infrastructure/host-integrations/host-integrations-list/lighttpd-integration.mdx

```diff
@@ -112,64 +116,68 @@ integrations:
     # interval: 30s
     config:
       name: lighttpdFlex
-      global:
-        base_url: http://lighttpd.net/
       apis:
-        - event_type: status
-          url: status?format=plain
-        - event_type: {'<THE_PLACE_WHERE_METRICS_ARE_CAPTURED>'}
-          commands:
-            - cache: status?format=plain
-              split_by: ": "
-          snake_to_camel: true
-
+        - event_type: <EVENT_TYPE>
+          url: <URL>
 
-  - name: nri-flex
-    interval: 10s
-    # timeout: 5s
-    config:
-      name: test
-      apis:
-        - event_type: {'<THE_PLACE_WHERE_METRICS_ARE_CAPTURED>'}
-          commands:
-            - run: curl 'http://lighttpd.net/status?auto'
-              split_by: ": "
-              regex_matches:
-                - expression: (\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)
 ```
 
 </Step>
+  
+<Step>
+  
+### Forward Lighttpd logs to New Relic
+ 
+You can use our [log forwarding](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Lighttpd logs to New Relic.
+
+On Linux machines, your log file named `logging.yml` should be present in the below path. If you don't see your log file, then you will need to create it by following the above log forwarding documentation.
+```
+Path: /etc/newrelic-infra/logging.d/
+```
+
+After creating the log file, add the below script to the `logging.yml` file:
+  
+<img width="330" alt="image" src="https://github.com/newrelic/docs-website/assets/101187392/9cabe7e7-4d92-4e82-8602-d8e94f2efe28">
```

Comment:
Please add the configuration text instead of images, so that the user can copy the snippet and use it in their configuration.